src/App.js

```diff
@@ -6,24 +6,12 @@ import NavBar from './components/NavBar';
 import Books from './components/Books';
```

Comment:
- [ ] I was unable to run your app because you have an import issue (see image). Your file name should be `Navbar`, not `NavBar` based on your file.

  You should change this line from
  
  ```
  import NavBar from './components/NavBar';
  ```
  to
  ```
  import NavBar from './components/Navbar';
  ```

  ![image](https://user-images.githubusercontent.com/36542986/214822603-f0405305-1d91-40b2-9c68-4199a0848387.png)